cli/tests/integration.rs

```diff
@@ -0,0 +1,10 @@
+#[path = "common/mod.rs"]
+mod common;
+
+#[cfg(feature = "end-to-end-tests")]
+#[path = "end_to_end/mod.rs"]
+mod end_to_end;
+
+#[cfg(feature = "extensions")]
+#[path = "extensions/mod.rs"]
+mod extensions;
```

Comment:
Are these path annotations necessary? I feel like they should load properly automatically?